include/odp/api/spec/ipsec.h

```diff
@@ -772,6 +772,76 @@ typedef struct odp_ipsec_sa_param_t {
 
 } odp_ipsec_sa_param_t;
 
+/** IPv4 address size */
+#define ODP_IP4ADDR_SIZE 4
+
+/** IPv6 address size */
+#define ODP_IP6ADDR_SIZE 16
+
+/**
+ * Retrieve IPSEC SA information
+ */
+typedef struct odp_ipsec_sa_info_t {
+	/** Copy of IPSEC Security Association (SA) parameters
+	 *
+	 * All pointers will be set to NULL.
```

Comment:
Maybe the context pointer should not be set to null? It is opaque data to ODP and could as well be returned.

I am not sure this this ponters-set-to-null should even be described here instead of as part of odp_ipsec_sa_info() documentation.

Then I wonder if inbound.lookup_param.dst_addr should be returned since also tunnel addresses are returned?

Reply:
> 
> 
> Maybe the context pointer should not be set to null? It is opaque data to ODP and could as well be returned.

Missed the context pointer. Agreed that it should be returned as it is.

> 
> I am not sure this this ponters-set-to-null should even be described here instead of as part of odp_ipsec_sa_info() documentation.

Agreed. Will move it to the odp_ipsec_sa_info() documentation.

> 
> Then I wonder if inbound.lookup_param.dst_addr should be returned since also tunnel addresses are returned?

Missed that. I will add buffer space for inbound.lookup_param.dst_addr as well.